src/molecules/Amount/Amount.native.js

```diff
@@ -122,15 +155,21 @@ const Amount = ({ size, testID, children, currency }) => {
 
 Amount.propTypes = {
   children: PropTypes.string.isRequired,
-  size: PropTypes.oneOf(['medium', 'large', 'xlarge', 'xxlarge', 'xxxlarge']),
+  size: PropTypes.oneOf(['xsmall', 'medium', 'large', 'xlarge', 'xxlarge', 'xxxlarge']),
   testID: PropTypes.string,
   currency: PropTypes.oneOf(geISOCurrencyList()),
+  variant: PropTypes.oneOf(['camel', 'normal', 'camel-subtle', 'normal-subtle']),
```

Comment:
Wait what?? We don't do composition in props
why `subtle`?

Reply:
If we keep allowing this then we'll end up scenarios like `camel-bold` `camel-bold-subtle` and so on and then everything goes for a toss

Reply:
Camel & normal have a subtle version too. Our initial API was supposed to have variants as subtle & normal but subtle somehow got missed during implementation.